crypto/txt_db/txt_db.c

```diff
@@ -14,6 +14,14 @@
 #include <openssl/buffer.h>
 #include <openssl/txt_db.h>
 
+#ifdef __GNUC__
+# pragma GCC diagnostic ignored "-Wunused-function"
+#endif
+#ifdef __clang__
+# pragma clang diagnostic ignored "-Wunused-function"
+#endif
```

Comment:
Why is this needed?

Reply:
because without it, clang generates an error. see https://github.com/openssl/openssl/blob/master/test/sparse_array_test.c#L21 for example.